server/Makefile

```
@@ -440,6 +438,16 @@ check-style: plugin-checker vet golangci-lint ## Runs style/lint checks
 gotestsum:
 	$(GO) install gotest.tools/gotestsum@v1.13.0
 
+install-ci-tools: gotestsum install-golangci-lint install-govet install-mockery install-mockgen install-morph install-msgp ## Install all tools required by CI
+
+go-cache: setup-go-work install-ci-tools ## Pre-warm Go module and build cache for CI
```

checkmake (0.3.2)

[warning] Target body for "go-cache" exceeds allowed length of 5 lines (6).

(maxbodylength)